### ISSUE TITLE
Liteloader mod not blocked tested with it.

### ISSUE THREAD

**Issue by Fonzee9** [Photon-GitHub/AACAdditionPro#60]
I blocked liteloader in config tried to join on my alt thats not opped and it didn't kick me :(

**Comment by Fonzee9** [comment]
Therefore a player can still use liteloader on my server and use the all light mod which basically makes it so you don't need torchs which is over powered to players that do.

**Comment by Photon-GitHub** [comment]
Can you please post your server log and your config here?

**Comment by Fonzee9** [comment]
This is what happens when my alt joins with Liteloader its as if it doesn't even know he has it.

19.04 09:26:47 [Server] INFO UUID of player SpiderMan31 is 2837834c-5052-44b8-a4f6-90b8fd41bff1
19.04 09:26:47 [Connect] User SpiderMan31, IP censored
19.04 09:26:47 [Server] INFO [GAListener] Player: SpiderMan31 has 0 votes
19.04 09:26:47 [Multicraft] SpiderMan31 ran command Message of the Day

**Comment by Fonzee9** [comment]
Config 

   # LiteLoader is alternative mod-loader to forge.
   # Many ghost clients exist for LiteLoader, therefore I recommend to block it.
   LiteLoader:
      # Do you want to enable the LiteLoader-Control
      # This cannot be blocked; I recommend to kick if desired
      enabled: true
      commands_on_detection:
         - "aackick {player} LiteLoader"

**Comment by Photon-GitHub** [comment]
Is this still valid @Fonzee9 ?

**Comment by Fonzee9** [comment]
Yeah @Photon-GitHub just tested with latest version of Liteloader on 1.12 on my 1.12 server still could join perfectly fine and i am not opped or have any bypass perms just to add that in.

**Comment by Photon-GitHub** [comment]
Please recheck the latest version.

**Comment by DracarysPro** [comment]
Still does not work, and now throws a console error on join: https://hastebin.com/acixinohuy.pl
Aacadditionpro 3.8.1
AAC 3.3.9
Protocollib 4.4.0 b390
Spigot 1.12.2

**Comment by Photon-GitHub** [comment]
The error had nothing to do with LiteLoader clientcontrol.

**Comment by Photon-GitHub** [comment]
Far future as I am not really sure on how to fix it.
